commit fae583199ec54703650bff4d54b234f45b12892a
Author: elcoosp <elcoosp@gmail.com>
Date:   Sat May 2 12:33:16 2026 +0200

    fix: test broken

diff --git a/crates/glyim-cli/tests/integration.rs b/crates/glyim-cli/tests/integration.rs
index 78fdc87..fb27bdd 100644
--- a/crates/glyim-cli/tests/integration.rs
+++ b/crates/glyim-cli/tests/integration.rs
@@ -1040,9 +1040,3 @@ main = () => {
 "#;
     assert_eq!(pipeline::run(&temp_g(src), None).unwrap(), 0);
 }
-
-"#;
-    let full_src = format!("{}\n{}\n{}", vec_src, hashmap_src, main_code);
-    let input = temp_g(&full_src);
-    assert_eq!(pipeline::run(&input, None).unwrap(), 500);
-}
